src/components/views/messages/MVideoBody.tsx

```diff
@@ -143,16 +143,15 @@ export default class MVideoBody extends React.PureComponent<IProps, IState> {
     }
 
     async _videoOnPlay() {
-        const autoplay = SettingsStore.getValue("autoplayGifsAndVideos") as boolean;
-        if (autoplay || this.state.decryptedUrl || this.state.fetchingData || this.state.error) {
-            // The video has or will have the data.
+        const content = this.props.mxEvent.getContent();
```

Comment:
This should always be defined for unencrypted meida, or for encrypted media blobs. It will be undefined if we haven't got the blob yet.

Reply:
Why did you need to move `content`? It's okay to do it, but just it doesn't seem like it's used any earlier that it was before, so I am just wondering if I am missing something...

Reply:
Oh, I was trying a thing and then I decided not to do that. I can move it back for the sake of making the diff smaller, onesec..